mmv1/products/cloudfunctions/CloudFunction.yaml

```diff
@@ -118,6 +118,10 @@ properties:
     name: 'serviceAccountEmail'
     output: true
     description: 'The email of the service account for this function.'
+  - !ruby/object:Api::Type::String
+    name: 'buildServiceAccount'
```

Comment:
This YAML file is [excluded from generation](https://github.com/GoogleCloudPlatform/magic-modules/pull/11100/files#diff-6330766290dce11fbe6f233175b8cf216f13e9eafb28c75675d398b4e359c7d0R47), I believe it is just around for IAM generation. Either way, adding this doesn't actually do anything but also doesn't really hurt.

Reply:
acked